packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/options/components/activation-status/activation-status.html

```diff
@@ -16,6 +16,7 @@
     <div
         class="d-inline-block"
         data-ng-repeat="datacenter in ::$ctrl.datacentersState track by $index"
+        data-ng-if="datacenter.state === $ctrl.ACTIVATION_STATUS.enabled.name"
```

Comment:
You could also add a filter `| filter: {state: ACTIVATION_STATUS.enabled.name}` instead of `ng-if`